deeplearning4j-nn/src/main/java/org/deeplearning4j/nn/conf/layers/LSTM.java

```diff
@@ -0,0 +1,84 @@
+/*-
+ *
+ *  * Copyright 2015 Skymind,Inc.
+ *  *
+ *  *    Licensed under the Apache License, Version 2.0 (the "License");
+ *  *    you may not use this file except in compliance with the License.
+ *  *    You may obtain a copy of the License at
+ *  *
+ *  *        http://www.apache.org/licenses/LICENSE-2.0
+ *  *
+ *  *    Unless required by applicable law or agreed to in writing, software
+ *  *    distributed under the License is distributed on an "AS IS" BASIS,
+ *  *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  *    See the License for the specific language governing permissions and
+ *  *    limitations under the License.
+ *
+ */
+
+package org.deeplearning4j.nn.conf.layers;
+
+import lombok.*;
+import org.deeplearning4j.nn.api.Layer;
+import org.deeplearning4j.nn.api.ParamInitializer;
+import org.deeplearning4j.nn.conf.NeuralNetConfiguration;
+import org.deeplearning4j.nn.params.GravesLSTMParamInitializer;
+import org.deeplearning4j.nn.params.LSTMParamInitializer;
+import org.deeplearning4j.optimize.api.IterationListener;
+import org.deeplearning4j.util.LayerValidation;
+import org.nd4j.linalg.activations.IActivation;
+import org.nd4j.linalg.activations.impl.ActivationSigmoid;
+import org.nd4j.linalg.api.ndarray.INDArray;
+
+import java.util.Collection;
+import java.util.Map;
+
+/**
+ * LSTM recurrent net, based on Graves: Supervised Sequence Labelling with Recurrent Neural Networks
+ * http://www.cs.toronto.edu/~graves/phd.pdf
+ */
+@Data
+@NoArgsConstructor
+@ToString(callSuper = true)
+@EqualsAndHashCode(callSuper = true)
+public class LSTM extends AbstractLSTM {
+
+    private double forgetGateBiasInit;
+    private IActivation gateActivationFn = new ActivationSigmoid();
+
+    private LSTM(Builder builder) {
+        super(builder);
+        this.forgetGateBiasInit = builder.forgetGateBiasInit;
+        this.gateActivationFn = builder.gateActivationFn;
+    }
+
+    @Override
+    public Layer instantiate(NeuralNetConfiguration conf, Collection<IterationListener> iterationListeners,
+                    int layerIndex, INDArray layerParamsView, boolean initializeParams) {
+        LayerValidation.assertNInNOutSet("GravesLSTM", getLayerName(), layerIndex, getNIn(), getNOut());
```

Comment:
Yep, good catch, I'll fix that now. And re: backward compatibility: separate layers (i.e., GravesLSTM and LSTM will co-exist) so there shouldn't be any issues there.

Open to suggestions on the naming - though it seems like the no-peephole model (ie., this one) is in a sense more standard these days. Hence my thinking with just calling it "LSTM".